demo/src/app/components/+dropdown/dropdown-section.list.ts

```diff
@@ -82,6 +83,15 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/disabled-menu/disabled-menu.html'),
         outlet: DemoDropdownDisabledComponent
       },
+      {
+        title: 'Disabled item',
+        anchor: 'disabled-item',
+        component: require('!!raw-loader?lang=typescript!./demos/disabled-item/disabled-item.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/disabled-item/disabled-item.html'),
+        description: `<p>Add a <code>disabled</code> class to <code>&lt;a&gt;</code> for bootstrap 4 to make it disabled.</p>
```

Comment:
any reason to repeat same text twice?

Reply:
![joxi_screenshot_1519390484136](https://user-images.githubusercontent.com/21618857/36595089-8f649872-18a9-11e8-867e-a8e8261a4475.png)